src/ducks/transactions/TransactionActions.jsx

```diff
@@ -58,7 +58,14 @@ export const SyncTransactionActions = ({
   compact,
   className
 }) => (
-  <span className={cx('u-dib', className)}>
+  <span
+    className={cx(
+      {
+        'u-dib': !isModalItem
```

Comment:
Nit: When there is only one class i want to add conditionally, I prefer `cx(!isModalItem ? 'u-dib' : null, className)`